browser/src/control/Control.Toolbar.js

```diff
@@ -1067,10 +1067,11 @@ function setupToolbar(e) {
 
 	if (map.options.wopi && L.Params.closeButtonEnabled && !window.mode.isMobile()) {
 		$('#closebuttonwrapper').css('display', 'block');
-		$('#closebuttonwrapper').prop('title', _('Close document'));
-		map.uiManager.enableTooltip($('#closebuttonwrapper'));
+		$('#closebutton').prop('title', _('Close document'));
+		map.uiManager.enableTooltip($('#closebutton'));
 	} else if (!L.Params.closeButtonEnabled) {
 		$('#closebuttonwrapper').hide();
+		$('#closebuttonwrapperseparator').hide();
 	} else if (L.Params.closeButtonEnabled && !window.mode.isMobile()) {
 		$('#closebuttonwrapper').css('display', 'block');
```

Comment:
don;t we need display block here too?
possibly the default should be hidden state so we will not show it on mobile... have to check